gem/lib/earlgrey/configure_earlgrey.rb

```diff
@@ -353,7 +354,21 @@ def add_earlgrey_framework(target, framework_ref)
     # rubocop:disable Style/PredicateName
     def has_swift?(target)
       target.source_build_phase.files_references.any? do |ref|
-        SWIFT_FILETYPE == (ref.last_known_file_type || ref.explicit_file_type)
+        file_type = ref.last_known_file_type
+        if file_type.to_s.empty?
+          file_type = ref.explicit_file_type
+        end
+
+        if SWIFT_FILETYPE == file_type
+          return true
+        elsif OBJC_FILETYPE == file_type
```

Comment:
I think this actually doesn't make sense if you look at the method name. you can have a separate method like has_objc?, but I still don't know if you really need to check that.

also throwing the error doesn't seem to be a good idea here, has_swift? returns false if it doesnt have swift files.